### ISSUE TITLE
ruby 2.7 warnings

### ISSUE THREAD

**Issue by zenspider** [piotrmurach/strings#9]
Tons of warnings coming up when I test our gem against ruby 2.7:

```
.../strings-0.1.7/lib/strings.rb:19: warning: The last argument is used as the keyword parameter
.../strings-0.1.7/lib/strings/align.rb:39: warning: for `align' defined here
.../strings-0.1.7/lib/strings/align.rb:42: warning: The last argument is used as the keyword parameter
.../strings-0.1.7/lib/strings/align.rb:85: warning: for `align_center' defined here
.../strings-0.1.7/lib/strings.rb:19: warning: The last argument is used as the keyword parameter
.../strings-0.1.7/lib/strings/align.rb:39: warning: for `align' defined here
.../strings-0.1.7/lib/strings/align.rb:42: warning: The last argument is used as the keyword parameter
.../strings-0.1.7/lib/strings/align.rb:85: warning: for `align_center' defined here
```

probably more, with the output above limited by our usage of the gem.

**Comment by zenspider** [comment]
I figured out that I named the travis setup wrong... it should be `2.7.0-preview2`.

**Comment by piotrmurach** [comment]
Released new version `0.1.8` with your fix.
